_posts/2024-01-14-House of Iranian culture in Bangladesh.md

```diff
@@ -0,0 +1,23 @@
+---
+title: خانه فرهنگ ایران در بنگلادش 
+tags: [آموزش زبان فارسی حضوری, آموزش زبان فارسی الکترونیکی همزمان, تالیف کتاب, برگزاری آزمون تعیین سطح]
+date: 2024-01-14 13:50:00 +0330
+image: 
+    path: /assets/img/logos/House of Iranian culture in Bangladesh.jpg
```

Comment:
It would be better not to use any space in image names to avoid possible URL problems.